scripts/person/person.gd

```diff
@@ -1118,7 +1131,12 @@ func get_birth_amount_name():
 func get_race_display():
 	var rvar = ''
 	if race != race_display:
-		rvar = ' (' + race_display + ')'
+		if race_display in globals.dictionary.hybriddict:
+			var temp1 = "[color=yellow]Subspecies: "
+			var temp2 = "[/color]"
+			rvar = ' (' + temp1 + race_display + temp2 + ')'
```

Comment:
temp1 and temp2 appear to be entirely unnecessary and their strings can be combined with the parenthesis

Reply:
Fixed